roles/wazuh/ansible-wazuh-manager/defaults/main.yml

```diff
@@ -260,6 +260,26 @@ wazuh_manager_config:
     - server: null
       port: null
       format: null
+  integrations:
+    - name: 'slack'
```

Comment:
The ` roles/wazuh/ansible-wazuh-manager/templates/var-ossec-etc-ossec-server.conf.j2` does render the template if the variables are defined. 

I think the variables should be defined to null by default to avoid rendering blocks in `ossec.conf` that are not activated or properly configured.